web/rest/brand/config/api/raw/provider.yml

```
@@ -342,6 +342,7 @@ Ivoz\Provider\Domain\Model\WebPortal\WebPortal:
       - urlType
 Ivoz\Provider\Domain\Model\WebPortal\Logo: ~
 
+Ivoz\Provider\Domain\Model\Company\Invoicing: ~
```

This change will permit every operation over Invoicing Entity, is this the intention? or we want to load only Invoicing Model to use on some parent Entity?

```suggestion
Ivoz\Provider\Domain\Model\Company\Invoicing:
  itemOperations: []
  collectionOperations: []
```

---

Invoicing is not an entity, just a VO. Like Logo, multilang descriptions, etc.